lifecycle/src/lifecycle_talker.cpp

```diff
@@ -154,7 +152,7 @@ class LifecycleTalker : public rclcpp_lifecycle::LifecycleNode
     // Overriding this method is optional, a lot of times the default is enough.
     LifecycleNode::on_activate(state);
 
-    RCUTILS_LOG_INFO_NAMED(get_name(), "on_activate() is called.");
+    RCLCPP_INFO(get_logger(), "on_activate() is called.");
```

Comment:
I think it's better to remove https://github.com/ros2/demos/blob/1a5f69e95dc60ad289e5faa85bf80ce08cc21002/lifecycle/src/lifecycle_talker.cpp#L30?

Reply:
Is it necessary to create another PR to remove https://github.com/ros2/demos/blob/0cf803e739120af898a00ad7dadeaafce58f273c/lifecycle/src/lifecycle_service_client.cpp#L27 and https://github.com/ros2/demos/blob/d9c8dacaeddbb7fd8316b1e2bd9b04160f8c6a2c/lifecycle/src/lifecycle_listener.cpp#L22?

Reply:
Excellent points.

I've gone ahead and removed them in e54b9af.  That also caused me to reexamine some of the dependencies here, and clean things up just a smidge.  So now this should be a lot more complete.